fix C syntax error and field name in conditional netdev ops struct, 
triggering on kernel >= 2.6.29 and CONFIG_NET_POLL_CONTROLLER=y.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/driver/wrapndis.c
+++ b/driver/wrapndis.c
@@ -1744,7 +1744,7 @@ static const struct net_device_ops ndis_
 	.ndo_set_mac_address = ndis_set_mac_address,
 	.ndo_get_stats = ndis_get_stats,
 #ifdef CONFIG_NET_POLL_CONTROLLER
-	.poll_controller = ndis_poll_controller;
+	.ndo_poll_controller = ndis_poll_controller,
 #endif
 };
 #endif


